client/my-sites/upgrades/domain-management/edit/card/header/index.jsx

```diff
@@ -21,16 +24,15 @@ const Header = React.createClass( {
 	},
 
 	render() {
-		const domain = this.props.domain;
+		const { domain, isDomainOnly } = this.props;
 
-		if ( ! domain ) {
+		if ( ! domain || isDomainOnly ) {
```

Comment:
@aidvu @drewblaisdell  Removing the badge is enough IMHO. I agree that the header may be redundant, but we should probably consider any changes to Calypso-wide screens very carefully. @aidvu How about starting a separate issue for reviewing the IA of the domain management screen?